locale/en/LC_MESSAGES/django.po

```diff
@@ -333,7 +333,7 @@ msgstr ""
 msgid ""
 "\n"
 "          This event is in the <strong>Anonymous Voting</strong> mode, all "
-"the speakers are hidden for voting, you should base your vote just in the "
+"the speakers are hidden for voting, you should base your vote just on the "
```

Comment:
How about **for** --> **during** ?...

Reply:
Using "during" is so much better in this case.